.docs/content/1.concepts/10.plugins.md

```diff
@@ -0,0 +1,47 @@
+# Using external plugins with ArmoniK.Core
+
+## Queue adaptors as external plugins
+
+We have the possibility to have a queue adaptor as an external project and to use it as a plugin to ArmoniK.Core when needed. More about adaptors is available [here](https://github.com/aneoconsulting/ArmoniK.Core/blob/main/.docs/content/1.concepts/7.adaptors.md). Using a queue adaptor as an external plugin was possible thanks to dynamic loading of adaptors, and the use of docker images.
+
+### Loading queue adaptor into a C# project
+
+ArmoniK.Core is based on C# projects where queue adaptors are dynamically loaded. Dynamic assembly loading allows to create dotnet applications with plugins. We will refer to the application to which the plugins will be connected as the hosting application.
```

Comment:
ArmoniK.Core is based on C# projects where queue adaptors are loaded dynamically. Dynamic assembly loading allows to create dotnet applications with plugins. The hosting application refers to the application connected to the plugins.